### ISSUE TITLE
K-9 crashing on selecting recipients (beta)

### ISSUE THREAD

**Issue by dschuermann** [open-keychain/open-keychain#2364]
from beta feedbkack:

> Since actual Beta i cant send any Email with k9. As soon as i enter the 
> second Letter of a Receiver adress k9 Shut down. I have tested actual k9 
> with non-beta openkeychain and also tried uninstall openkeychain. Both 
> worked without any Problem. Huawei P9 Lite actual emui 5.0.1. with Android 7

**Comment by wiktor-k** [comment]
I did get something like this with 5.2-beta2:

```
java.lang.RuntimeException: An error occurred while executing doInBackground()
       at android.os.AsyncTask$3.done(AsyncTask.java:353)
       at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
       at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
       at java.util.concurrent.FutureTask.run(FutureTask.java:271)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
       at java.lang.Thread.run(Thread.java:764)
Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'int java.lang.CharSequence.length()' on a null object reference
       at android.text.util.Rfc822Tokenizer.tokenize(Rfc822Tokenizer.java:51)
       at android.text.util.Rfc822Tokenizer.tokenize(Rfc822Tokenizer.java:173)
       at com.fsck.k9.mail.Address.<init>(Address.java:50)
       at com.fsck.k9.mail.Address.<init>(Address.java:41)
       at com.fsck.k9.view.RecipientSelectView$Recipient.<init>(RecipientSelectView.java:600)
       at com.fsck.k9.activity.compose.RecipientLoader.fillContactDataFromCursor(RecipientLoader.java:377)
       at com.fsck.k9.activity.compose.RecipientLoader.fillContactDataFromCursor(RecipientLoader.java:332)
       at com.fsck.k9.activity.compose.RecipientLoader.fillContactDataFromNameAndEmail(RecipientLoader.java:324)
       at com.fsck.k9.activity.compose.RecipientLoader.fillContactDataFromQuery(RecipientLoader.java:260)
       at com.fsck.k9.activity.compose.RecipientLoader.loadInBackground(RecipientLoader.java:137)
       at com.fsck.k9.activity.compose.RecipientLoader.loadInBackground(RecipientLoader.java:27)
       at android.content.AsyncTaskLoader.onLoadInBackground(AsyncTaskLoader.java:315)
       at android.content.AsyncTaskLoader$LoadTask.doInBackground(AsyncTaskLoader.java:69)
       at android.content.AsyncTaskLoader$LoadTask.doInBackground(AsyncTaskLoader.java:64)
       at android.os.AsyncTask$2.call(AsyncTask.java:333)
       at java.util.concurrent.FutureTask.run(FutureTask.java:266)
       ... 3 more
```

**Comment by Valodim** [comment]
The original bug from the beta feedback was fixed in eda75276ceb800fd10ec7f27977be84d384279c8. The additional stack trace looks like a k9 problem.
